dev-python/sphinx-prompt/metadata.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+  <maintainer type="project">
+    <email>python@gentoo.org</email>
+    <name>Python</name>
+  </maintainer>
+  <upstream>
+    <remote-id type="pypi">flake8</remote-id>
+    <remote-id type="gitlab">pycqa/flake8</remote-id>
```

Comment:
These two lines look like a copy and paste mistake.

Reply:
Correct, thanks, will fix it.